# current time (sec)
# previous time (sec)
# Both are needed to be able to compute REAL WORLD DELTA t
float32 current_time
float32 previous_time
